receiver/otlpreceiver/json.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 The OpenTelemetry Authors
```

Comment:
IIRC the year was removed from the license, but `make addlicense` still adds it.